Trivia Maze/src/application/Main.java

```diff
@@ -450,10 +448,7 @@ else if(x >= COLS || x < 0)
 	
 	public static void setVerticalArchway(int y, int x, ArchwayStatus status)
 	{
-		if (stage == null || gameMaze == null)
-			return;
-		
-		else if(status == null)
+		if(status == null)
```

Comment:
Should also check if verticalArchways is null.

Reply:
Those are two separate methods.